WordPress/Classes/ViewRelated/Site Creation/Web Address/DesignSystem/Components/PrimaryButton.swift

```diff
@@ -0,0 +1,36 @@
+import SwiftUI
+
+public struct PrimaryButton: View {
```

Comment:
[Suggestion] This is a good use case of the `buttonStyle` modifier. See how it works https://www.hackingwithswift.com/quick-start/swiftui/customizing-button-with-buttonstyle

Reply:
Afaik `ButtonStyle` is mostly used for the interaction states. But beyond that, I want to limit the UI customization in a Design System mentality. It would further scale for button types with images or certain input values we require. Then the API/View would guide the call site to what to provide and expect rather than being a configuration tool